vtStorManaged/IRunTimeDll.h

```diff
@@ -16,21 +16,21 @@ limitations under the License.
 </License>
```

Comment:
Added IRunTimeDll to be able to get the desired function from the run time linked module.